grasa_event_locator/templates/editEvent.html

```diff
@@ -296,7 +304,11 @@ <h5 class="modal-title" id="exampleModalLabel">Submit</h5>
             }else{
                 var name = list[i]
             }
-            $( "."+location).append( "<option value=\""+list[i]+"\">"+list[i]+"</option>" );
+            var selectedText = ''
+            if (selected.includes(list[i])){
+              selectedText=' selected'
+            }
+            $( "."+location).append( "<option value=\""+list[i]+"\""+selectedText+">"+list[i]+"</option>" );
         }
     }
```

Comment:
@ldidonato @leiyinmon If you have a chance to review this JavaScript later, please take a look.